bundles/org.openhab.binding.pentair/src/main/resources/OH-INF/thing/intellichlor.xml

```diff
@@ -7,38 +7,101 @@
 	<thing-type id="intellichlor">
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="ip_bridge"/>
-			<bridge-type-ref id="pentair_serial_bridge"/>
+			<bridge-type-ref id="serial_bridge"/>
 		</supported-bridge-type-refs>
 
 		<label>Intellichlor IC40</label>
 		<description>Pentair Intellichlor IC40</description>
 
 		<channels>
-			<channel id="saltoutput" typeId="saltoutput"/>
+			<channel id="saltOutput" typeId="saltOutput"/>
 			<channel id="salinity" typeId="salinity"/>
+			<channel id="ok" typeId="ok"/>
+			<channel id="lowFlow" typeId="lowFlow"/>
+			<channel id="lowSalt" typeId="lowSalt"/>
+			<channel id="veryLowSalt" typeId="veryLowSalt"/>
+			<channel id="highCurrent" typeId="highCurrent"/>
+			<channel id="cleanCell" typeId="cleanCell"/>
+			<channel id="lowVoltage" typeId="lowVoltage"/>
+			<channel id="lowWaterTemp" typeId="lowWaterTemp"/>
 		</channels>
 
-		<config-description>
-			<parameter name="id" type="integer" required="false">
-				<label>ID</label>
-				<description>The ID of the device (in decimal, not hex)</description>
-				<default>96</default>
-			</parameter>
-		</config-description>
+		<properties>
+			<property name="version">Version</property>
+			<property name="model">Model</property>
+		</properties>
+
+		<representation-property>id</representation-property>
 	</thing-type>
 
-	<channel-type id="saltoutput">
-		<item-type>Number</item-type>
+	<channel-type id="saltOutput">
+		<item-type>Number:Dimensionless</item-type>
 		<label>Salt Output (%)</label>
 		<description>Current salt output setting for the chlorinator (%).</description>
 		<state readOnly="true"/>
```

Comment:
Don;t you want to specify the unit pattern ? 
UoM/units have been subject to change since this PR was created,

Reply:
In reviewing the documentation, it still indicates a Percent unit is a Dimensionless type.  Maybe I'm not understanding the feedback.

Reply:
I referred to the `state` tag. That tag can have a `pattern` attribute. 
Als provided a suggestion with the `unitHint` attribute
```suggestion
		<item-type unitHint="%">Number:Dimensionless</item-type>
		<state readOnly="true" pattern="%.0f %unit%" />
```

Reply:
I did try to add the unitHint to the item-type tag, but get an error in Eclipse - "Element 'item-type' is a simple type, so it cannot have attributes".  My guess is I have to refer to a new Schema?  I won't have a chance to look at this until next weekend.

Reply:
Im not using eclipse, but vscode. I remember something similar and had to refresh/clear some cache. This might help.